plugins/single_plugins/resize.cpp

```diff
@@ -249,6 +250,11 @@ class wayfire_resize : public wf::plugin_interface_t
                 view->set_moving(false);
             view->set_resizing(false);
             end_wobbly(view);
+
+            view_change_viewport_signal workspace_may_changed;
+            workspace_may_changed.view = this->view;
+            workspace_may_changed.old_viewport_invalid = false;
```

Comment:
what about setting current viewport? We know the viewport that we're moving to (current one)

Reply:
That was the reason I omitted these, it made only sense to me in a scenario where all workspaces need to be recalculated and setting the current view-port probably adds a byte somewhere, that's why the tile -plugin was adjusted to not react if old_viewport_invalid=true